package.json

```diff
@@ -22,12 +22,12 @@
   },
   "devDependencies": {
     "@grafana/aws-sdk": "0.0.37",
-    "@grafana/data": "8.0.5",
+    "@grafana/data": "8.5.10",
     "@grafana/e2e": "8.1.2",
     "@grafana/e2e-selectors": "8.1.2",
-    "@grafana/runtime": "8.0.5",
-    "@grafana/toolkit": "8.0.5",
-    "@grafana/ui": "8.0.5",
+    "@grafana/runtime": "8.5.10",
+    "@grafana/toolkit": "8.5.10",
+    "@grafana/ui": "8.5.10",
```

Comment:
Updates made because in `datasource.ts` we override the query function so we can customize how we run the query. Updated these so that we get access to `getRef()` in order to set the datasource ref on the query when it's run.

As a result of these updates to the dev deps, the `plugin.json` file updates the minimum Grafana dependency to v8.5.x.